### PR TITLE
Fix style

### PR DESCRIPTION
**Pull request by LauraCourquin**

### CODE REVIEW

**Review by Hamza-Luciano — APPROVED** [review]
modification of style

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/Logo/style.jsx [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import styled from "styled-components";
 
 export default styled.div`
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin-top: 46vh;
+  z-index: 1;
```
Comment (LoicBrassart) [inline frontend/src/components/Logo/style.jsx:9]: Meh

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**